suites/quincy/rbd/tier-2_rbd_regression.yaml

```diff
@@ -76,6 +76,19 @@ tests:
       polarion-id: CEPH-83573758
 
   - test:
+      config:
```

Comment:
Since you are using all default configurations, I think you can rebase code from master to fetch Vasi's changes of adding default values and remove this config from test